tests/bgp/test_bgpmon.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+import logging
+from netaddr import IPAddress, IPNetwork
+import ptf.testutils as testutils
+from jinja2 import Template
+import ptf.packet as scapy
+from ptf.mask import Mask
+from tests.common.fixtures.ptfhost_utils import change_mac_addresses      # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import remove_ip_addresses       # lgtm[py/unused-import]
+from tests.common.helpers.generators import generate_ips as generate_ips
+
+pytestmark = [
+    pytest.mark.topology('any'),
+]
+
+BGPMON_TEMPLATE_FILE = 'bgp/bgpmon.j2'
+BGPMON_CONFIG_FILE = '/tmp/bgpmon.json'
+BGP_PORT = 179
+BGP_CONNECT_TIMEOUT = 120
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture
+def common_setup_teardown(duthost, ptfhost):
+    mg_facts = duthost.minigraph_facts(host=duthost.hostname)['ansible_facts']
+    peer_addr = generate_ips(1, "%s/%s" % (mg_facts['minigraph_vlan_interfaces'][0]['addr'],
```

Comment:
As I remember we need to generate a random ip address and check that 'ip route get <ip>' will return default route